comment-sample/package.json

```diff
@@ -13,7 +13,6 @@
 	"engines": {
 		"vscode": "^1.35.0"
```

Comment:
The vscode engine version should also be increased to the version that the API proposal was finalized in.

Reply:
Since it's not obvious what version that is, we can just use 1.65, since that one is definitely ok.

Reply:
Thanks. You're right, it wasn't obvious to me what version you were referring to. I had changed the `"@types/vscode"` to use `~` so the stable type definitions would match that engine version since it was just getting the latest one before and it looked like they were all in the v1.35 type definitions. I'll change it to 1.65.